### ISSUE TITLE
Calendar Pro Visual custom version does not support

### ISSUE THREAD

**Issue by SKmudasali** [okviz/calendar-pro-issues#3]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The custom version dose not support while changing system type to custom or other apart from Gregorian.
The QT its showing Deaccenting order Q3,Q4, Q1. 
and i would like to change Finance year start month November and end month October. test file shared and snap enclosed please look into this.
[Test.zip](https://github.com/okviz/calendar-pro-issues/files/9626521/Test.zip)

![image](https://user-images.githubusercontent.com/107244827/191767501-e0f6097c-6217-4d77-8d80-703e5547bd32.png)

### Power BI

Power BI Desktop

### Sample report

![test](https://user-images.githubusercontent.com/107244827/191769357-634660bb-69aa-45a0-8f78-c3e5e33ab325.PNG)

### Screenshot

![image](https://user-images.githubusercontent.com/107244827/191769248-e0cbd51f-0103-487e-b84a-1138de93d1ca.png)

**Comment by danieleperilli** [comment]
Sorry, can you rephrase? I opened you files but I didn't understand the problem. Do you want the quarter order is not re-calculated on the starting month?
